detector/calorimeter/HCalTileBarrel_o1_v01_geo.cpp

```diff
@@ -40,55 +41,66 @@ static dd4hep::Ref_t createHCal(dd4hep::Detector& lcdd, xml_det_t xmlDet, dd4hep
   xml_comp_t xSteelSupport = xmlDet.child(_Unicode(steel_support));
   double dSteelSupport = xSteelSupport.thickness();
 
-  lLog << MSG::DEBUG << "steel support thickness: " << dSteelSupport << " [cm]" << endmsg;
+  lLog << MSG::DEBUG << "steel support thickness (cm): " << dSteelSupport / dd4hep::cm  << endmsg;
   lLog << MSG::DEBUG << "steel support material:  " << xSteelSupport.materialStr() << endmsg;
 
   double sensitiveBarrelRmin = xDimensions.rmin() + xFacePlate.thickness() + space;
 
   // Hard-coded assumption that we have two different sequences for the modules
   std::vector<xml_comp_t> sequences = {xmlDet.child(_Unicode(sequence_a)), xmlDet.child(_Unicode(sequence_b))};
```

Comment:
You can check if the xmlDet has the childs instead of the check in line 52
```
x_slice.hasAttr(_U(sequence_a)) && x_slice.hasAttr(_U(sequence_b))
```